### ISSUE TITLE
parsing wrong winlog.event_data.CallTrace.keyword

### ISSUE THREAD

**Issue by gnomefish** [Neo23x0/sigma#598]
Hi there,

I noticed this detection doesn't work, however if you  manually change "winlog.event_data.CallTrace.keyword" to "winlog.event_data.CallTrace" - is this a bug in how sigma is outputting the Kibana query?

python3 sigmac -I -t es-qs -c /opt/sigma/tools/config/winlogbeat.yml /opt/sigma/rules/windows/sysmon/sysmon_lsass_memdump.yml 
(winlog.channel:"Microsoft\-Windows\-Sysmon\/Operational" AND winlog.event_id:"10" AND winlog.event_data.TargetImage:"C\:\\windows\\system32\\lsass.exe" AND winlog.event_data.GrantedAccess:"0x1fffff" AND winlog.event_data.CallTrace.keyword:(*dbghelp.dll* OR *dbgcore.dll*))

Cheers,
Luk

**Comment by gnomefish** [comment]
Same happens with the dcsync rule, also .keyword breaking the query

**Comment by neu5ron** [comment]
no, this was a major change in Elastic 7.x that changed the entire functionality of keyword fields. .keyword no longer exists. Since it was not announced and not everyone on Elastic 7.x the default is still to use .keyword.
You can use the sigmac option to disable the `.keyword ` option via `keyword_blacklist` e.g. `sigmac -Okeyword_blacklist=*`

let me know if this helps

**Comment by crimsoncore** [comment]
yes that's it :) thanks!!!

**Comment by crimsoncore** [comment]
did something change? I'm sure this worked a while ago :

sigmac -Okeyword_blacklist=* -t es-qs -c /opt/sigma/tools/config/winlogbeat.yml /opt/sigma/rules/netview.yaml

and the resulting query still has the .keyword in it:

(winlog.channel:"Microsoft\-Windows\-Sysmon\/Operational" AND winlog.event_id:"1" AND 
winlog.event_data.CommandLine.keyword:(*\\net *\\net1))

**Comment by neu5ron** [comment]
nothing has been changed, there is a PR to make things more controllable by user. but for right now this should still work. Try maybe with quote? like so:

`-O keyword_blacklist="*"`

**Comment by frack113** [comment]
Sorry this post is closed automatically because it is not more active
